common/src/main/java/tachyon/util/io/FileUtils.java

```diff
@@ -100,13 +100,13 @@ public static void changeLocalFileToFullPermission(String filePath) throws IOExc
    * If the sticky bit of the 'file' is set, the 'file' is only writable to its owner and the owner
    * of the folder containing the 'file'.
    *
-   * @param file absolute file path
+   * @param file file path to set the sticky bit
    */
   public static void setLocalFileStickyBit(String file) {
     try {
       // sticky bit is not implemented in PosixFilePermission
       if (file.startsWith(TachyonURI.SEPARATOR)) {
-        Runtime.getRuntime().exec("chmod o+t " + file);
+        Runtime.getRuntime().exec("chmod o+t " + new File(file).getAbsolutePath());
```

Comment:
It's not a bug, previously method `setLocalFileStickyBit` will receive **absolute file path** as parameter, I'd like to **relaxing this restriction** by add `getAbsolutePath` in the method body so that caller may pass in **relative or absolute file path**.
-  If the passed in parameter `file` is already an `absolute path`, `new File(file).getAbsolutePath()` will just simply return the string itself. 
- If the passed in parameter is a `relative` one, `new File(file).getAbsolutePath()` will get absolute file path by simplifying combine the **. (the working directory)** where you run your program with 'file' to get the absolute path.

Here the parameter name(it's previous) is a bit confusing, i will improve it.